api/src/main/java/org/openmrs/api/impl/PersonServiceImplTest.java

```diff
@@ -0,0 +1,158 @@
+package org.openmrs.api.impl;
+
+import org.junit.Assert;
+
+/**
+ * Created by ira on 3/8/17.
+ */
+public class PersonServiceImplTest {
```

Comment:
I am sorry about all of this. I am wrestling really hard wit Git and working across 2 computers. The auto generation tool created that extra class and I copied the code onto the main class for the real code. I am not sure  how the test code generation tool works.